contracts/issuer/IssuerAccessFacet.sol

```diff
@@ -86,6 +87,8 @@ contract IssuerAccessFacet is AIssuerFacet, IHasMembers {
     );
     // Keep track of the governorship.
     LibIssuerAccess.data().fastGovernorships[governor].add(msg.sender, false);
+
+    emit GovernorshipAdded(msg.sender, governor);
```

Comment:
I would suggest that this event is changed so that it includes "which FAST" it is targetting, as per what we were discussing.
For example: `GovernorshipAdded(msg.sender, governor, fast)`.
This way, the subscriber can maintain off-chain structures without having to query the chain right after this event is emitted, or having to subscribe to each FAST.